File: problem-2/problem-2.php
<?php 		
	$sum = 2;	
	$a = 1;
	$b = 2 ;
	$c = 3;
	while ( $c < 4000000 ) {
		$c = $a + $b;
        $a = $b;
		$b = $c;
		$rest = fmod( $c, 2 );
         if ( $rest == 0 ) {
			$sum += $c;
		}
	}
	echo $sum;
?>

File: problem-2/problem-2.c
#include <stdio.h>
 
main() {
	int a = 1,b = 2,c = 3,sum = 2,rest;
	while (c < 4000000 ) {
		c = a + b;
		a = b;
		b = c;
		rest = c % 2; 
		if (rest == 0) {
			sum = sum + c;
		}
	}
	printf( "The integer i am looking for is %d", sum ) ;
	
	return 0;
}


